src/charm.py

```diff
@@ -260,7 +215,7 @@ def _configure_service(
 
     def _get_srsenb_command(self) -> str:
         """Returns srs enb command."""
-        srsenb_command = [SRS_ENB_BINARY]
+        srsenb_command = ["/snap/bin/srsran.srsenb"]
```

Comment:
Do you think it would make sense to run eNB as `root` as well? I know it's not necessary, but if we do, it shouldn't complain about being unable to run a privileged thread (or something like that).